src/widgets/about-video/ui/about-video.tsx

```diff
@@ -1,10 +1,13 @@
+import classNames from 'classnames/bind';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './about-video.scss';
+import styles from './about-video.module.scss';
+
+const cx = classNames.bind(styles);
 
 type AboutVideoProps = { lang?: 'en' | 'ru' };
 
-const localizedContent = {
+export const localizedContent = {
   en: { title: 'RS School video' },
   ru: { title: 'Видео RS School' },
 };
```

Comment:
It could be better to move this to a separate file, but I'm not sure, and I'm going to discuss this with the team.